CalculateTotal.js

```diff
@@ -0,0 +1,42 @@
+function getOrderTotal(items) {
+  var total = 0;
+  items.map((x) => {
+    total += x.cost * x.quantity;
+  });
+
+  // console.log("total", total);
+  return total;
+}
+
+var calculateSalesTax = function(total) {
+  return total + (total * 15) / 100;
+};
```

Comment:
There has been inconsistent use of function expressions and function statements in the code. 

```suggestion
function calculateSalesTax(total) {
  return total + (total * 15) / 100;
}
```